pajbot/apiwrappers/seventv.py

```diff
@@ -78,38 +60,23 @@ def get_global_emotes(self, force_fetch=False):
             force_fetch=force_fetch,
         )
 
-    def fetch_channel_emotes(self, channel_name):
+    def fetch_channel_emotes(self, channel_id):
         """Returns a list of channel-specific 7TV emotes in the standard Emote format."""
-        query_string = """
-query fetchUserEmotes($id: String!) {
-    user(id: $id) {
-        emotes {
-            id
-            name
-            width
-            height
-        }
-    }
-}"""
-
-        params = {
-            "query": query_string,
-            "variables": {
-                "id": channel_name,
-            },
-        }
-        response = self.post("gql", json=params)
+        try:
+            response = self.get(f"users/twitch/{channel_id}")
+        except HTTPError as e:
+            if e.response.status_code == 404:
+                # user does not have any 7TV emotes
+                return []
 
-        if response["data"]["user"] is None:
-            # user does not have any 7TV emotes
-            return []
+            raise e
 
-        return self.parse_emotes(response["data"]["user"]["emotes"])
+        return self.parse_emotes(response["emote_set"]["emotes"])
 
-    def get_channel_emotes(self, channel_name, force_fetch=False):
+    def get_channel_emotes(self, channel_id, force_fetch=False):
         return self.cache.cache_fetch_fn(
-            redis_key=f"api:7tv:channel-emotes:{channel_name}",
```

Comment:
While extremely unlikely, it's possible for a collision to occur between old and new data schemas here.
I think that's an acceptable risk/imperfection though, you basically have to manufacture that setup

I just wanted to point it out